core/block.py

```
@@ -0,0 +1,31 @@
+import time
+
+class Block:
+    def __init__(self, index, previous_hash, transactions, timestamp=None):
```

Ruff (0.15.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)